src/components/ResetPassword/ResetPasswordForm.jsx

```diff
@@ -41,17 +43,17 @@ const ResetPasswordForm = () => {
     navigate('/login');
   };
 
-  // TODO: Implement reset password
-  const resetPassword = () => {
-    // Make request to reset password here
-    // eslint-disable-next-line no-console
-    console.log('password reset complete');
+  const resetPassword = async data => {
+    try {
+      await confirmNewPassword(code, data.newPassword);
+    } catch (err) {
+      // TODO: replace with toast component
+      console.log(err.message);
+    }
   };
 
   const onSubmit = data => {
-    resetPassword();
-    // eslint-disable-next-line no-alert
-    alert(JSON.stringify(data));
+    resetPassword(data);
     onOpenResetModal();
```

Comment:
It looks like they changed the Figma design to be a confirmation page instead of a Modal, so would it be possible to implement that here?

Reply:
Duron created a common confirmation page and updated all the modals to use this as needed in https://github.com/ctc-uci/find-your-anchor-frontend/pull/90